.github/workflows/fuzz.yml

```diff
@@ -16,10 +16,6 @@ concurrency:
 env:
   CARGO_TERM_COLOR: always
 
-defaults:
-  run:
-    working-directory: openmls
```

Comment:
What about the `working-directory`s in the other workflows? We could update them as well and operate on the workspace instead.

Reply:
Done. I left that for another PR since it's not easy for me to trigger CI jobs. I kept feature parity because I would like to improve the CI in another PR. For example currently, evercrypt tests are not executed in the CI and I think they should